wpiutil/src/llvm/raw_ostream.cpp

```diff
@@ -845,12 +845,12 @@ raw_null_ostream::~raw_null_ostream() {
 #endif
 }
 
-void raw_null_ostream::write_impl(const char *Ptr, size_t Size) {
+void raw_null_ostream::write_impl(const char * /*Ptr*/, size_t /*Size*/) {
```

Comment:
Sure, done.

https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions suggests /**/, but I'm happy to do whatever you guys want.